operator/pkg/ciliumendpointbatch/manager_test.go

```diff
@@ -175,6 +175,27 @@ func TestDeepCopyCeps(t *testing.T) {
 		m.updateCebInCache(Ceb, true)
 		ceb, _ = m.getCebFromCache(cn)
 		assert.Equal(t, ceb.DeepEqual(Ceb), true, "Local CEB should match with CEB in datastore")
+
+		m1 := newCebManagerIdentity(newQueue(), 2)
+		m1.InsertCepInCache(cep1)
+		cn = m1.InsertCepInCache(cep2)
```

Comment:
I think we should have a new test case where you insert CEPs of "different" identityID.